lib/modules/platform/types.ts

```diff
@@ -190,7 +194,7 @@ export interface Platform {
   addReviewers(number: number, reviewers: string[]): Promise<void>;
   addAssignees(number: number, assignees: string[]): Promise<void>;
   createPr(prConfig: CreatePRConfig): Promise<Pr | null>;
-  getRepos(): Promise<string[]>;
+  getRepos(param?: AutodiscoverConfig): Promise<string[]>;
```

Comment:
```suggestion
  getRepos(config?: AutodiscoverConfig): Promise<string[]>;
```
whoops, wrong suggestion

Reply:
That was just me missing that instance of "param".